### PR TITLE
Moving Agent SDK Tests to System Application tests.

### PR DESCRIPTION
**Pull request by JesperSchulz**
#### Summary 
Moving the Agent SDK Tests to the System Application tests, where they belong. This is step 1 of the move operation.

#### Work Item(s) 
Fixes [AB#629464](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/629464)

### CODE REVIEW

File: src/System Application/Test/Agent SDK/app.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    },
+    {
+      "id": "9856ae4f-d1a7-46ef-89bb-6ef056398228",
+      "name": "System Application Test Library",
+      "publisher": "Microsoft",
+      "version": "$(app_minimumVersion)"
+    },
+    {
+      "id": "5d86850b-0d76-4eca-bd7b-951ad998e997",
+      "name": "Tests-TestLibraries",
```
Comment (JesperSchulz) [inline src/System Application/Test/Agent SDK/app.json]: I don't think this is a valid dependency in the System Application.

File: src/System Application/Test/Agent SDK/app.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
+{
+  "id": "a8c7f9d2-4e5b-4a1c-9f8e-6d3b2a1c4e5f",
+  "name": "Agent SDK Tests",
+  "publisher": "Microsoft",
+  "brief": "Comprehensive test suite for validating Agent SDK functionality.",
+  "description": "The Agent SDK Tests app provides extensive automated testing for the Agent SDK functionality. Includes unit tests for agent operations, API validation, and agent lifecycle management. Features test coverage for error scenarios, warning conditions, successful operations, and edge cases to ensure reliable agent SDK workflows.",
+  "version": "$(app_currentVersion)",
```
Comment (mazhelez) [inline src/System Application/Test/Agent SDK/app.json]: This doesn't work with AL-Go yet.

File: src/System Application/Test/Agent SDK/app.json (hunk 1 above)
Comment (mazhelez) [reply src/System Application/Test/Agent SDK/app.json]: No, it is not.

Comment (JesperSchulz) [reply src/System Application/Test/Agent SDK/app.json]: Removing dependency for now. Refactoring will follow.

File: src/System Application/Test/Agent SDK/app.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    },
+    {
+      "id": "dd0be2ea-f733-4d65-bb34-a28f4624fb14",
+      "name": "Library Assert",
+      "publisher": "Microsoft",
+      "version": "29.0.0.0"
+    },
+    {
+      "id": "5afe217e-507a-40b4-9aa5-f4325b6e8230",
+      "name": "Agent Test Library",
```
Comment (qutreson) [inline src/System Application/Test/Agent SDK/app.json:28]: This app is not in BCApps yet, maybe it has to be moved first, before moving this one.

Comment (JesperSchulz) [reply src/System Application/Test/Agent SDK/app.json:28]: Then why did it compile locally? Ok, something to look into after easter 🐰

File: src/System Application/Test/Agent SDK/app.json [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      "id": "e7320ebb-08b3-4406-b1ec-b4927d3e280b",
+      "publisher": "Microsoft",
+      "name": "Any",
+      "version": "29.0.0.0"
+    }
+  ],
+  "screenshots": [],
+  "platform": "29.0.0.0",
+  "application": "29.0.0.0",
+  "resourceExposurePolicy": {
```
Comment (JesperSchulz) [inline src/System Application/Test/Agent SDK/app.json:47]: Cannot take this dependency either:
```suggestion
  "platform": "29.0.0.0",
  "resourceExposurePolicy": {
```

File: src/System Application/Test/Agent SDK/app.json (hunk 3 above)
Comment (qutreson) [reply src/System Application/Test/Agent SDK/app.json:28]: Looks like the build system agrees with me:

image
